server/googleActions.js

```diff
@@ -301,6 +306,7 @@ GoogleActions = {
 				createdAt  : obj.data.createdTime,
 				updatedAt  : obj.data.modifiedTime,
 				lastViewed : obj.data.properties.lastViewed,
+				pageCount  : obj.data.properties?.pageCount || 1,
```

Comment:
```suggestion
				pageCount : obj.data.properties.pageCount,
```
Same comments as above.